jsdoc/tag-defs/access.js

```diff
@@ -1,13 +1,16 @@
-module.exports = function(accessTagTransform, trimWhitespaceTransform) {
-  var name = 'access'
-  accessTagTransform.addTag(name);
-  accessTagTransform.addValue('private');
-  accessTagTransform.addValue('protected');
-  accessTagTransform.addValue('public');
-
+module.exports = function(createDocMessage) {
+  STANDARD_VALUES = ['private', 'protected', 'public'];
   return {
-    name: name,
-    docProperty: 'access',
-    transforms: [trimWhitespaceTransform, accessTagTransform]
+    name: 'access',
+    transforms: function(doc, tag, value) {
+      var value = value.trim();
+      if (STANDARD_VALUES.indexOf(value) !== -1) {
```

Comment:
I see. I would rather not be too restrictive. One can always write a processor that can check the values of the `.access` property later, which doesn't limit others from having custom access values.